packages/rpc/src/methods/stacks/stx-transfer-stx.ts

```diff
@@ -1,20 +1,14 @@
 import { z } from 'zod';
 
 import { defineRpcEndpoint } from '../../rpc/schemas';
-import {
-  baseStacksTransactionConfigSchema,
-  stacksTransactionDetailsSchema,
-} from './_stacks-helpers';
+import { stacksTransactionDetailsSchema } from './_stacks-helpers';
 
 export const stxTransferStx = defineRpcEndpoint({
   method: 'stx_transferStx',
-  params: z.intersection(
-    z.object({
-      recipient: z.string(),
-      amount: z.coerce.number(),
-      memo: z.string().optional(),
-    }),
-    baseStacksTransactionConfigSchema
-  ),
+  params: z.object({
+    recipient: z.string(),
+    amount: z.coerce.number(),
+    memo: z.string().optional(),
+  }),
```

Comment:
let's use 

```suggestion
  params: z.object({
    recipient: z.string(),
    amount: z.coerce.number(),
    memo: z.string().optional(),
  }).passthrough(),
```

which allows other props, without this, we'd be introducing a worse breaking change